samza-core/src/main/java/org/apache/samza/task/StreamOperatorTask.java

```diff
@@ -77,8 +77,9 @@ public StreamOperatorTask(StreamGraphBuilder graphBuilder) {
 
   @Override
   public final void init(Config config, TaskContext context) throws Exception {
+    ApplicationRunner runner = ApplicationRunner.fromConfig(config);
```

Comment:
Issue: This should probably be passed in from SamzaContainer via the constructor.